set.mm

```diff
@@ -17151,13 +17151,19 @@ disjoint variable condition ( ~ sb2 ).  Theorem ~ sb6f replaces the
        23-Jul-2023.) $)
     equsb3 $p |- ( [ x / y ] y = z <-> x = z ) $=
       ( vw weq equequ1 sbievw2 ) BCEACEDCEBADBDCFDACFG $.
-  $}
 
-  ${
-    $d y z $.
     $( Substitution applied to the atomic wff with equality.  Variant of
        ~ equsb3 .  (Contributed by AV, 29-Jul-2023.) $)
     equsb3r $p |- ( [ x / y ] z = y <-> z = x ) $=
+      ( vw weq equequ2 sbievw2 ) CBECAECDEBADBDCFDACFG $.
+  $}
+
+  ${
+    $d y z $.
+    $( Obsolete version of ~ equsb3 as of 2-Sep-2023.  (Contributed by AV,
+       29-Jul-2023.)  (Proof shortened by Wolf Lammen, 2-Sep-2023.)
```

Comment:
(Proof shortened by Wolf Lammen, 2-Sep-2023.) should be added to the comment of ~equsb3r!?

Reply:
Of course.